ansible/roles/edu-sharing/tasks/edu-sharing-docker/custom_menu_entries.yml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
This (and also a lot of the other files in this directory) seems to be a copy of the old edu-sharing-config-tasks (with just some little path-adjustments). So, now we have (nearly) duplicated files - which makes it hard to maintain the ansible-scripts. Better to use a general approach for old and new edu-sharing with just some individual path adjustments.